src/main/java/seedu/storemando/logic/commands/SortQuantityCommand.java

```diff
@@ -20,10 +29,17 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(MESSAGE_NO_ITEMS_TO_SORT);
         }
 
-        ItemComparatorByQuantity comparator = new ItemComparatorByQuantity();
-        model.updateSortedItemList(comparator);
-        model.setItems(model.getSortedItemList());
-        return new CommandResult(MESSAGE_SUCCESS);
+        if (this.isAscending) {
```

Comment:
I think u just need the first line to be in if else statement, the rest should be outside as one right? :)